src/intervals/trigonometric.jl

```diff
@@ -27,15 +27,15 @@ end
 function sin{T}(a::Interval{T})
     isempty(a) && return a
 
-    whole_range = Interval(-one(T), one(T))
+    whole_range = Interval{T}(-1, 1)
```

Comment:
My original motivation for these changes was that I was suspecting that they were introducing a type instability problem (which I doubt, but I just wanted to make sure).

I also think that the `Interval{T}(-1, 1)` is easier to read.

However, now that you pointed it out, I went digging, and found that 
`@edit one(Float64)`
gives the following definition:
```
one{T<:Number}(::Type{T}) = convert(T,1)
```
So it's doing the conversion anyway ;)

Also if you define 
```
f{T}(::Type{T}) = Interval(-one(T), one(T))
g{T}(::Type{T}) = Interval{T}(-1, 1)
```
and do `@code_native f(Float64)` and similarly for `g`, you'll find that there are fewer instructions for `g`. So all round, the new version seems to be a win!